lib/exchange/provider/thorchain_exchange.provider.dart

```diff
@@ -206,8 +208,35 @@ class ThorChainExchangeProvider extends ExchangeProvider {
     );
   }
 
+  static Future<Map<String, String>?>? lookupAddressByName(String name) async {
+    final uri = Uri.https(_baseURL, '$_nameLookUpPath$name');
+    final response = await http.get(uri);
+
+    if (response.statusCode != 200) {
+      throw Exception('Unexpected HTTP status: ${response.statusCode}');
```

Comment:
just return null so it continues checking other address resolvers

Reply:
done